### PR TITLE
Update free-programming-books-et.md

### PR DESCRIPTION
**Pull request by oskar-anderson**
## What does this PR do?
Add resource(s) | Remove resource(s)

### Description
Add Taltech material, remove dead link, redirect Python material link to Gitlab (migrated).

### Why is this valuable (or not)?
More content, fix existing content

### For book lists, is it a book? For course lists, is it a course? etc.
'Andmebaaside alused' is technically a course, but the material reads more like a book. Rest should be OK.

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md)
- [x] Search for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction)

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: books/free-programming-books-et.md [hunk 1]
```diff
@@ -1,18 +1,24 @@
 ### Index
 
+* [Algorithms and Data Structures](#algo)
```
Comment (LuigiImVector) [inline books/free-programming-books-et.md:3]:
```suggestion
* [Algorithms & Data Structures](#algorithms--data-structures)
```

In the other files it is written like this

File: books/free-programming-books-et.md [hunk 2]
```diff
@@ -1,18 +1,24 @@
...
 * [PHP](#php)
 * [Python](#python)
 * [R](#r)
 * [SQL](#sql)
 * [WebGL](#webgl)
 
 
+### <a id="algo"></a>Algorithms and Data Structures
+
+* [Algoritmid ja andmestruktuurid (2003, third edition)](https://dspace.ut.ee/bitstream/handle/10062/16872/9985567676.pdf) - Jüri Kiho (PDF)
```
Comment (LuigiImVector) [inline books/free-programming-books-et.md:19]:
```suggestion
### Algorithms & Data Structures

* [Algoritmid ja andmestruktuurid (2003, 3rd edition)](https://dspace.ut.ee/bitstream/handle/10062/16872/9985567676.pdf) - Jüri Kiho (PDF)
```

File: books/free-programming-books-et.md [hunk 3]
```diff
@@ -25,24 +31,23 @@
 
 ### Java
 
-* [Java õppematerjalid](https://ained.ttu.ee/javadoc/index.html) - TTÜ õppematerjalid
+* [Java õppematerjalid](https://javadoc.pages.taltech.ee) - TTÜ (Taltech) õppematerjalid
```
Comment (LuigiImVector) [inline books/free-programming-books-et.md:34]:
```suggestion
* [Java õppematerjalid](https://javadoc.pages.taltech.ee) - TalTech
```

I think "taltech" is more appropriate

File: books/free-programming-books-et.md [hunk 4]
```diff
@@ -25,24 +31,23 @@
...
 #### AngularJS
 
-* [AngularJS raamistiku õppematerjal](http://www.cs.tlu.ee/teemad/get_file.php?id=400) - Tallinna Ülikooli, Sander Leetus (PDF)
+* [AngularJS raamistiku õppematerjal](https://www.cs.tlu.ee/teemad/get_file.php?id=400) - Tallinna Ülikool, Sander Leetus (PDF)
 
 
 #### Vue
 
-* [Vue](http://www.cs.tlu.ee/teemaderegister/get_file.php?id=715) - Tallinna ülikooli õppematerjal
+* [Vue](https://www.cs.tlu.ee/teemaderegister/get_file.php?id=715) - Tallinna Ülikooli õppematerjal
```
Comment (LuigiImVector) [inline books/free-programming-books-et.md:50]:
```suggestion
* [Vue.js raamistiku õppematerjal](https://www.cs.tlu.ee/teemaderegister/get_file.php?id=715) - Tallinna Ülikooli, Fred Korts (PDF)
```

File: books/free-programming-books-et.md [hunk 5]
```diff
@@ -55,15 +60,17 @@
 * [Programmeerimise õpik](https://progeopik.cs.ut.ee) - Tartu Ülikooli Arvutiteaduse Instituut
 * [Pythoni algteadmised](https://courses.cs.ut.ee/MTAT.03.100/2012_fall/uploads/opik/00_eessona.html) - Tartu Ülikooli Arvutiteaduse Instituut
 * [Pythoni wikiraamat](https://et.wikibooks.org/wiki/Python) - Wikiõpikud
+* [Pythoni õppematerjalid](https://pydoc.pages.taltech.ee) - TTÜ (Taltech) õppematerjalid
```
Comment (LuigiImVector) [inline books/free-programming-books-et.md:63]:
```suggestion
* [Pythoni õppematerjalid](https://pydoc.pages.taltech.ee) - TalTech
```

File: books/free-programming-books-et.md [hunk 6]
```diff
@@ -55,15 +60,17 @@
...
 
 ### R
 
-* [Statistiline andmeteadus ja visualiseerimine R keele abil](https://andmeteadus.github.io/2015/rakendustarkvara_R/) - Mait Raag, Raivo Kolde
+* [Statistiline andmeteadus ja visualiseerimine R keele abil](https://andmeteadus.github.io/2015/rakendustarkvara_R) - Mait Raag, Raivo Kolde
 
 
 ### SQL
 
+* [Andmebaaside alused](https://enos.itcollege.ee/~priit/1.%20Andmebaasid/1.%20Loengumaterjalid) - Priit Raspel, TTÜ (Taltech) (HTML)
```
Comment (LuigiImVector) [inline books/free-programming-books-et.md:73]:
```suggestion
* [Andmebaaside alused](https://enos.itcollege.ee/~priit/1.%20Andmebaasid/1.%20Loengumaterjalid) - TalTech, Priit Raspel (HTML)
```

**Review by LuigiImVector — CHANGES_REQUESTED** [review]

File: books/free-programming-books-et.md (hunk 4 above)
Comment (eshellman) [reply books/free-programming-books-et.md:50]: We typically credit authors, not publishers. In some other entries, we use "Taltech" as a corporate author, not as the publisher.

File: books/free-programming-books-et.md (hunk 6 above)
Comment (eshellman) [reply books/free-programming-books-et.md:73]: just the author, please

File: books/free-programming-books-et.md [hunk 7]
```diff
@@ -55,15 +60,17 @@
...
 
 ### R
 
-* [Statistiline andmeteadus ja visualiseerimine R keele abil](https://andmeteadus.github.io/2015/rakendustarkvara_R/) - Mait Raag, Raivo Kolde
+* [Statistiline andmeteadus ja visualiseerimine R keele abil](https://andmeteadus.github.io/2015/rakendustarkvara_R) - Mait Raag, Raivo Kolde
 
 
 ### SQL
 
+* [Andmebaaside alused](https://enos.itcollege.ee/~priit/1.%20Andmebaasid/1.%20Loengumaterjalid) - TalTech,Priit Raspel (HTML)
```
Comment (eshellman) [inline books/free-programming-books-et.md:73]: Everything else looks good, what about this one?

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Pull request merged**